README.md

```diff
@@ -1,220 +1,172 @@
-![logo](https://owo.whats-th.is/8mT5kxc.png)
-##### Project rewrite coming soon™
-[![Build Status](https://travis-ci.com/Pierce01/MinecraftLauncher-core.svg?branch=master)](https://travis-ci.com/Pierce01/MinecraftLauncher-core)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-![version](https://img.shields.io/badge/stable_version-3.18.1-blue)
+![Main Logo](/imgs/header.png)
+
+[![MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+![Version](https://img.shields.io/badge/stable_version-3.19.0-blue)
 
 MCLC (Minecraft Launcher Core) is a NodeJS solution for launching modded and vanilla Minecraft without having to download and format everything yourself.
 Basically a core for your Electron or script based launchers.
 
 ### Getting support
-I've created a Discord server for anyone who needs to get in contact with me or get help!
-<p>
-   <a href="https://discord.gg/8uYVbXP">
-   <img src="https://img.shields.io/discord/568550848871923723?logo=discord"
-      alt="chat on Discord"></a>
-<p>
+
+Do you need to get in contact with me or get help? Join my Discord Server
+
+[![Discord](https://img.shields.io/discord/568550848871923723?logo=discord)](https://discord.gg/8uYVbXP)
 
 ### Installing
 
-`npm i minecraft-launcher-core`
+```bash
+# npm
+npm i minecraft-launcher-core
 
-### Standard Example
-```javascript
-const { Client, Authenticator } = require('minecraft-launcher-core');
-const launcher = new Client();
+# Yarn
+yarn add minecraft-launcher-core
+
+# pnpm
+pnpm add minecraft-launcher-core
+```
+
+### Example
+
+```js
+import { Authenticator, Client } from 'minecraft-launcher-core';
 
-let opts = {
-    // For production launchers, I recommend not passing 
-    // the getAuth function through the authorization field and instead
-    // handling authentication outside before you initialize
-    // MCLC so you can handle auth based errors and validation!
-    authorization: Authenticator.getAuth("username", "password"),
-    root: "./minecraft",
+const launcher = new Client();
+const options = {
```

Comment:
What about a `defineConfig` (like the one you can see in vitejs) to ensure what's passed in the options ?